main.tf

```diff
@@ -1,13 +1,74 @@
-resource "random_integer" "example" {
+#-----------------------------------------------------------------------------------------------------------------------
+# Subscribe the Acccount to Security Hub
+#-----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_account" "this" {
   count = module.this.enabled ? 1 : 0
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Optionally subscribe to Security Hub Standards 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards.html
+#
+# arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0
+# arn:aws:securityhub:us-east-1::standards/aws-foundational-security-best-practices/v/1.0.0
+# arn:aws:securityhub:us-east-1::standards/pci-dss/v/3.2.1
+#-----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_standards_subscription" "this" {
+  for_each      = local.enabled_standards_arns
+  depends_on    = [aws_securityhub_account.this]
+  standards_arn = each.key
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Optionally configure Event Bridge Rules and SNS subscriptions 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-integration-types.html
+#-----------------------------------------------------------------------------------------------------------------------
+module "sns_topic" {
+  source = "git::https://github.com/cloudposse/terraform-aws-sns-topic.git?ref=tags/0.9.0"
+  count  = local.create_sns_topic ? 1 : 0
+
+  context         = module.this.context
+  subscribers     = {}
+  sqs_dlq_enabled = false
+}
+
+resource "aws_cloudwatch_event_rule" "imported_findings" {
+  count       = local.enable_notifications == true ? 1 : 0
+  name        = "${module.this.id}-imported-findings"
+  description = "SecurityHubEvent - Imported Findings"
+  tags        = module.this.tags
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
-  }
+  event_pattern = jsonencode(
+    {
+      "source" : [
+        "aws.securityhub"
+      ],
+      "detail-type" : [
+        "Security Hub Findings - Imported"
+      ]
+    }
+  )
 }
 
+resource "aws_cloudwatch_event_target" "imported_findings" {
+  count     = local.enable_notifications == true ? 1 : 0
+  rule      = aws_cloudwatch_event_rule.imported_findings[0].name
+  target_id = "SendToSNS"
```

Comment:
Any reason we might want to parameterize this with a `variable`?

Reply:
Actually, the default is to generate a unique id, so no reason to set it ourselves. Removed

c083990a030e8209bdae88e660ff9ce1f1f4a9c8